plugins/include/amxmisc.inc

```diff
@@ -869,3 +869,32 @@ stock get_players_ex(players[MAX_PLAYERS], &num, GetPlayersFlags:flags = GetPlay
 	get_flags(_:flags, strFlags, charsmax(strFlags));
 	get_players(players, num, strFlags, team);
 }
+
+/**
+ * Registers a chat command that can be executed via both "say" and "say_team".
+ *
+ * @note For a list of possible access flags, see the ADMIN_* constants in
+ *       amxconst.inc
+ * @note Opting in to FlagManager enables the admin privileges to be overwritten
+ *       by the end user via the cmdaccess.ini config file.
+ * @note Automatic detection for FlagManager will only include a command if it
+ *       has required privileges (flags is not -1) and it is not a command
+ *       starting with "say".
+ *
+ * @param client_cmd    Command to register
+ * @param function      Callback function
+ * @param flags         Admin privilege flags required
+ * @param info          Command description
+ * @param FlagManager   0 opts out of flag manager, 1 opts in, -1 selects
+ *                      automatically
+ * @param info_ml       If true, the parameter "info" will be looked up as multilingual key
+ *
+ * @noreturn
```

Comment:
TODO: return command index.

Reply:
@OciXCrom do you think it is possible? There are two commands registered inside.

Reply:
@WPMGPRoSToTeMa I'm pretty sure the second index will always be the first index minus one, so we can just return the index of the first one and state in the documentation how to retreive the second one.

Reply:
That's honestly not intuitive, better return them by ref, optionally. Other than that, you shouldn't ever have to do such arithmetic, it's closely tied to the implementation details - if they change (for example we start using handles instead) it will break.

Also commands register by AMXX have indices in negative - if one command is -10, the next one is -11. So the order is reversed.

Reply:
@rsKliPPy You're right. I changed it to store them by ref.

Btw I know about the reversed order, that's why I made it return the "say_team" index and use +1 to get the "say" index.

Reply:
Oh sorry, I was tripping it said "- 1" for some reason.